articles/getting-started/the-basics.md

```diff
@@ -103,7 +103,7 @@ In our example, `ExampleCo` wants their users to be able to login with username/
 1. Enable all three connections for the mobile app
 
 ::: note
-For more information on the supported identity providers, refer to [Identity Providers Supported by Auth0](/identityproviders). For details on how to enable a connection for an application, refer to [Application Connections](/applications/connections).
+For more information on the supported identity providers, refer to [Identity Providers Supported by Auth0](/identityproviders). For details on how to enable a connection for an application, refer to [Connections](/connections).
```

Comment:
`/identityproviders` and `/connections` are the same page

Reply:
Yes, but they have been the same thing for a long time (and the applications/connections doc that this link is replacing was more of the same). Also, identityproviders and connections won't be the same very shortly when my other sub-task on Connections moves out--it's one of the issues I've identified and resolved. When I do my search/replace on these links for that PR, I want to be able to locate this and keep the nuance. It's not introducing any additional problems in the docs, so I left it alone for the time being since I know it will be resolved soon.